crates/flipperzero/examples/storage.rs

```diff
@@ -23,10 +25,11 @@ entry!(main);
 
 fn main(_args: *mut u8) -> i32 {
     // First, we'll create a file on the SD card and write "Hello, Rust!" to it.
+    let path = CStr::from_bytes_with_nul(b"/ext/hello-rust.txt\0").unwrap();
```

Comment:
We should use the `sys::c_string!` macro here (it automatically adds a b"\0").

Reply:
Good to know. Thanks.

Reply:
Oh, sorry. I don't know how I missed that. This is fine.